extension/view/js/TranslationNotificationManager.js

```diff
@@ -93,4 +93,8 @@ class TranslationNotificationManager {
         const message = { command: "displayStatistics", tabId: this.tabId };
         this.bgScriptListenerCallback(message);
     }
+
+    isPageLanguageSupported(){
+        return this.languageSet.has(this.detectedLanguage);
```

Comment:
We should check that language pair page-lang->user-lang or page-lang->en and en->user-lang are supported. Models of opposite directions are completely decoupled from each other and we can support only one-way translation.

Reply:
After giving this a second thought there's some assumptions we are making about this issue for free that I'm not sure if they are right:

- We are putting to much trust that the cld module will return right language, and we already know how naive that module is and its flaws (like confusing es and pt very often)
- We are assuming that the portion of the page's content to be analyzed to always be representative of the majority of the page's language.

So I think this is more complicated than just preventing the infobar of displaying when the supposed page's language do not match what exists in the model registry. Before that we'll need to review the cld and the page's language analysis first and also decide if we want to actually prevent the infobar of displaying or not (maybe we want to always display it and only don't auto select the language we think it was detected? 

In a normal world that would be a UI + product decision, but since we know we don't have that, we'll need to make a decision ourselves.

Reply:
I think we should definitely work on improving cld accuracy but I don't think always displaying the infobar will be useful. Our language coverage is pretty limited atm and displaying just to show that we don't support the page language will be annoying for users. I would suggest assuming cld works correctly and relying on its output here, abstract from it and work on this issue separately.